worlds/ffmq/__init__.py

```diff
@@ -69,20 +70,20 @@ def __init__(self, world, player: int):
         super().__init__(world, player)
 
     def generate_early(self):
-        if self.multiworld.sky_coin_mode[self.player] == "shattered_sky_coin":
-            self.multiworld.brown_boxes[self.player].value = 1
-        if self.multiworld.enemies_scaling_lower[self.player].value > \
-                self.multiworld.enemies_scaling_upper[self.player].value:
-            (self.multiworld.enemies_scaling_lower[self.player].value,
-             self.multiworld.enemies_scaling_upper[self.player].value) =\
-                (self.multiworld.enemies_scaling_upper[self.player].value,
-                 self.multiworld.enemies_scaling_lower[self.player].value)
-        if self.multiworld.bosses_scaling_lower[self.player].value > \
-                self.multiworld.bosses_scaling_upper[self.player].value:
-            (self.multiworld.bosses_scaling_lower[self.player].value,
-             self.multiworld.bosses_scaling_upper[self.player].value) =\
-                (self.multiworld.bosses_scaling_upper[self.player].value,
-                 self.multiworld.bosses_scaling_lower[self.player].value)
+        if self.options.sky_coin_mode == "shattered_sky_coin":
+            self.options.brown_boxes.value = 1
+        if self.options.enemies_scaling_lower.value > \
+                self.options.enemies_scaling_upper.value:
```

Comment:
This is short enough to be on one line now, no? :)

Same with a lot of the following lines

Reply:
+1
It's really hard to read. also the (..., ...) = (..., ...) assignment below is even worse with the breaks. If it needs breaks, there is no reason to do tuple assignments like this.

Reply:
Changed.